doc/source/sysadmin.rst

```diff
@@ -16,21 +16,21 @@ Debian wheezy / Debian jessie / Ubuntu
 
 Download and install the Public Signing Key for elasticsearch repo::
 
-    wget -qO - https://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
 
 Add elasticsearch repos in /etc/apt/sources.list.d/elasticsearch.list::
 
-    echo "deb http://packages.elasticsearch.org/elasticsearch/2.x/debian stable main" | sudo tee /etc/apt/sources.list.d/elasticsearch.list
+    echo "deb https://artifacts.elastic.co/packages/6.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-6.x.list
 
 Install requirements::
 
-    sudo apt-get update && sudo apt-get install python2.7 gcc python2.7-dev python-virtualenv openjdk-7-jre-headless elasticsearch
+    sudo apt-get update && sudo apt-get install python2.7 gcc python2.7-dev python-virtualenv apt-transport-https openjdk-8-jre-headless elasticsearch
```

Comment:
I don't think this will work. `apt-transport-https` must be installed _before_ `apt-get update`. So, yes, 3 invocations of `apt-get` are needed.